migrations/m000000_000000_install.php

```diff
@@ -330,6 +330,9 @@ public function up()
         $this->createIndex('class', models\TagAssign::tableName(), 'class');
         $this->createIndex('item_tag', models\TagAssign::tableName(), ['item_id', 'tag_id']);
 
+	    //CONTENT MODULE
```

Comment:
Started new branch for this feature.
https://github.com/boehsermoe/easyii/tree/feature/moduleMigration